os/os.proto

```diff
@@ -177,6 +202,10 @@ message TransferEnd {
 // The InstallResponse is used by the Target to inform the Client about the
 // state of the Install RPC. At any stage of the process the Target can reply
 // with an Error message which MUST terminate the stream.
+// Systems which install the image on all Supervisors in one command
+// will either have the image on both Supervisors or neither Supervisor.
+// Such systems do not need to sync the image from peer Supervisor and
+// will respond with TransferReady if the package is absent on the Supervisor.
```

Comment:
A dual Supervisor Target which requires installing the entire system with one Install RPC, will either have the image on both Supervisors or neither Supervisor. Such systems do not need to sync the image from the peer Supervisor and MUST respond with TransferReady if the package is absent from any Supervisor.

Reply:
this is missing

Reply:
done.